packages/components/src/base-control/style.scss

```diff
@@ -20,7 +20,3 @@
 		font-style: italic;
 	}
 }
-
-.components-base-control + .components-base-control {
```

Comment:
I wonder if we should remove that rule just like that. It has implications on all sidebars (and all other places where we use multiple controls in a row like the entities save modal) (multiple controls in a row) including in third-party blocks.

Reply:
I'd rather override it personally where we have horizontal controls today.

Reply:
I responded in https://github.com/WordPress/gutenberg/pull/20608#discussion_r390853392 — best I can tell it's either dead code or ill-advised. In all my web-inspecting, the rule was almost always overridden. 

The way to space base controls, I would suggest, is simple:

```
.components-base-control {
	margin-bottom: $grid-unit-20;
}
```

While you can achieve the same with the following:

```
.components-base-control + .components-base-control {
	margin-top: $grid-unit-20;
}
```

That's a far more roundabout way of doing it. And given margins collapse regardless, the former seems much more the correct way of spacing things.

Reply:
I guess the `.components-base-control .components-base-control__field` bottom padding makes this rule a bit redundunt. I'm actually ok removing it now.

Reply:
Should we remove the rule from the table block too? (it's overridden there)

Reply:
I'll remove it there and ship. Thanks for fast review!